vendor/gems/et_atos_export/app/services/et_atos_export/claim_file_builder/build_claim_text_file.rb

```diff
@@ -20,7 +20,7 @@ def self.render(claim)
           claim: claim, primary_claimant: claim.primary_claimant,
           primary_respondent: claim.primary_respondent,
           primary_representative: claim.primary_representative,
-          additional_respondents: claim.secondary_respondents
+          additional_respondents: claim.secondary_respondents.includes(:address)
```

Comment:
This was a performance issue when multiple claimants used.  Fixed to speed up the test suite as well as the app